### PR TITLE
apply yorkie-js-sdk v0.2.8

### PR DESCRIPTION
**Pull request by Eithea**
#### What this PR does / why we need it?

change interface name to apply : https://github.com/yorkie-team/yorkie-js-sdk/commit/47478e73a91634c15055adb1dfc0977e8656a5f0

#### What are the relevant tickets?

Fixes #

### Checklist
- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

**Comment by Eithea** [comment]
there is a bug about undo-redo but i could not find the cause.

**Comment by hackerwins** [comment]
@Eithea Thanks for the report. 
I couldn't reproduce the bug. Could you register the bug as a new issue?

File: webapp/package.json [hunk 1]
```diff
@@ -55,11 +55,13 @@
...
     "redux-thunk": "^2.3.0",
     "typescript": "^3.7.5",
-    "yorkie-js-sdk": "^0.2.7"
+    "yorkie-js-sdk": "^0.2.8"
   },
   "scripts": {
     "start": "REACT_APP_GIT_HASH=`git rev-parse --short HEAD` react-scripts start",
     "build": "REACT_APP_GIT_HASH=`git rev-parse --short HEAD` react-scripts build",
+    "start-win": "set REACT_APP_GIT_HASH=`git rev-parse --short HEAD` && react-scripts start",
+    "build-win": "set REACT_APP_GIT_HASH=`git rev-parse --short HEAD` && react-scripts build",
```
Comment (hackerwins) [inline webapp/package.json:64]:
```suggestion
    "start:win": "set REACT_APP_GIT_HASH=`git rev-parse --short HEAD` && react-scripts start",
    "build:win": "set REACT_APP_GIT_HASH=`git rev-parse --short HEAD` && react-scripts build",
```

File: webapp/src/components/Editor/DrawingBoard/index.tsx [hunk 2]
```diff
@@ -62,11 +62,11 @@ export default function DrawingBoard({ width, height }: { width: number; height:
 
     const clientId = client.getID()!;
     const handleUpdateMeta = (data: BoardMetadata) => {
-      const board = JSON.stringify(data);
+      const board = JSON.stringify(data || '');
       boardRef.current?.updateMetadata(clientId, {
         board,
       } as Metadata);
-      client?.updatePresence('board', board);
+      client?.updatePresence('board', board);
```
Comment (hackerwins) [inline webapp/src/components/Editor/DrawingBoard/index.tsx:69]:
```suggestion
      client?.updatePresence('board', board);
```

**Review by hackerwins — APPROVED** [review]
Thanks for your contribution.

**Pull request merged**